src/pfe/file-watcher/server/src/projects/constants.ts

```diff
@@ -36,3 +36,5 @@ export enum StartModes { run = "run", debug = "debug", debugNoInit = "debugNoIni
 export enum ControlCommands { stop = "stop", start = "start", restart = "restart" }
 
 export enum MavenFlags { profile = "-P ", properties = "-D " }
+
+export const serverLessAppPort = "-1";
```

Comment:
Suggest renaming this variable to something like `DisablePingPort` since "serverless" can be confused with serverless computing.